cli/src/utils/compareSchema.ts

```diff
@@ -127,3 +135,92 @@ export function compareSchemas(
 
   return { edits };
 }
+
+export const inferOldSchemaToNew = (oldSchema: Schemas.DBBranch): Schemas.BranchSchema => {
```

Comment:
This will be shared between frontend when https://github.com/xataio/frontend-next/issues/3873 is taken care of